extlib/scheduler-playground/lib/store.js

```diff
@@ -0,0 +1,38 @@
+var _ = require('underscore');
+
+var Store = function(options) {
+    // First and last points in the store
+    this.last  = new Date(options.last.getTime());
+    this.first = new Date(options.first.getTime());
+
+    // Points per second (data density)
+    this.density = options.density;
+
+    this.data = [];
+
+    this.init();
+};
+
+Store.prototype.init = function() {
+    var range = this.last.getTime() - this.first.getTime();
+    var maxPoints = (range / 1000) * this.density;
+
+    while (this.data.length < maxPoints) {
```

Comment:
Density is great parameter to be able to pass. Question: what happens when the maxPoints threshold is reached with all points representing of the first quarter of the time range requested?